pkg/types/compactor.go

```diff
@@ -46,7 +46,7 @@ type CompactorConfig struct {
 	DeltaSnapshotLeaseName string            `json:"deltaSnapshotLeaseName,omitempty"`
 	EnabledLeaseRenewal    bool              `json:"enabledLeaseRenewal"`
 	// see https://github.com/gardener/etcd-druid/issues/648
-	SleepFor wrappers.Duration `json:"sleepFor,omitempty"`
+	SleepForScrape wrappers.Duration `json:"sleepForScrape,omitempty"`
```

Comment:
There is no mention of "time" or "duration" in this field name. It simply says "sleep for scrape", which could be a boolean value for all we know, which could mean "if set to true, compactor will sleep for some scrape". There is no mention of what is being scraped, or what kind of value this field can take.

As mentioned in https://github.com/gardener/etcd-backup-restore/pull/660#discussion_r1308493167, please change this field name to make it more meaningful, for both developers and users of etcdbr.

Reply:
Done